README.md

```diff
@@ -32,14 +32,24 @@ Edit this document to include your answers after each question. Make sure to lea
 
 1. What is the DOM?
 
+The Document Object Model is an object representation of the HTML elements of the webpage. Within this interface we can manipulate and interact to change the functionality and design of the webpage. The DOM can be thought of as a bridge between the our code and the representation of it with it's functionality on our browsers.
+
 2. What is an event?
 
+Events are simply put, are the interactions users have with their webpage. This could be a number of things such as: mouse and keyboard inputs. The DOM creates and propagates these events and carries information about them conveying instructions as what should be done in each given case.
```

Comment:
Event don't necessarily have to be triggered by users. Load and transitionend come to mind